platform/feature-api/communication/remote-pod/src/remote.ts

```diff
@@ -331,7 +331,9 @@ export class RemoteServerPod implements ServerOf<PodEndpoint> {
         const { bubblewrapMiddlewarePort } = await import("@polypoly-eu/port-authority/dist/node");
         const [middleware, port] = bubblewrapMiddlewarePort(
             Bubblewrap.create(podBubblewrapClasses),
-            { limit: "10mb" }
+            {
```

Comment:
This is unrelated, right? Are you sure it's not better the original way?

Reply:
Well, Prettier wants it a certain way, right? If that's what it did...

Reply:
But that configuration has not changed, as far as I can tell. Not important, anyway.

Reply:
The Prettier works in mysterious ways, who are we to question The Prettier? :P